src/services/configService.js

```diff
@@ -15,6 +21,29 @@ export class ConfigService {
       .map(toDeviceGroupsModel);
   }
 
+  /** Creates a new device group */
+  static createDeviceGroup(payload) {
+    return HttpClient.post(`${ENDPOINT}devicegroups`, payload)
+      .map(toDeviceGroupModel);
+  }
+
+  static updateDeviceGroup(id, payload) {
+    return HttpClient.put(`${ENDPOINT}devicegroups/${id}`, payload)
+      .map(toDeviceGroupModel);
+  }
+
+  /** Delete a device group */
+  static deleteDeviceGroup(id) {
+    return HttpClient.delete(`${ENDPOINT}devicegroups/${id}`)
+      .map(_ => id);
+  }
+
+  /** Returns the account's device group filters */
+  static getDeviceGroupFilters() {
```

Comment:
nit: `getDeviceGroups`

Reply:
This is not loading the device groups. It is loading the filters used to make device groups.

Reply:
Ahh...get it now